protobufjs/src/server.ts

```diff
@@ -0,0 +1,61 @@
+import { Weather, Forecast, Wind } from "./weather";
+import * as grpc from "@grpc/grpc-js";
+import * as pbjs from "protobufjs";
+
+// Unfortunately protobufjs does not generate service definitions that is compatible with grpc.
```

Comment:
I'm using [@grpc/proto-loader's type generation](https://github.com/grpc/grpc-node/tree/master/packages/proto-loader#generating-typescript-types) to overcome this limitation.

@thesayyn @alexeagle If you're interested I can open a PR with that example.

Reply:
That'd be great. Does it generate the js files as well? We want full type checking coverage here.